data/sql/updates/pending_db_world/rev_1591646826613634700.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1591646826613634700');
+
+UPDATE `command` SET `help` = "Syntax: .reload page_text\r\nReload page_text table.\nYou need to delete your client cache in order to see the changes." WHERE `name` = "reload page_text";
```

Comment:
```suggestion
UPDATE `command` SET `help` = "Syntax: .reload page_text\r\nReload page_text table.\nYou have to delete your client cache or change the cache number in conf for your players to see the changes." WHERE `name` = "reload page_text";
```

i made the help clearer

can you add this to every string?

Reply:
What does that mean. Never heard of it?

Reply:
I agree with @BarbzYHOOL 's suggestion. @Kitzunu can you please edit this?

Changing the cache number will force all clients to reload their cache.